internal/pkg/meta/constants.go

```diff
@@ -23,4 +23,8 @@ const (
 	UserReserved2
 	// UserReserved3 is reserved for user-defined metadata.
 	UserReserved3
+	// UUIDRewrite stores the UUID that this machine will use instead of the one from the hardware.
+	UUIDRewrite
```

Comment:
```suggestion
	UUIDOverride
```

might be a better name?